src/unix/mod.rs

```diff
@@ -1712,3 +1724,26 @@ cfg_if! {
         pub use self::no_align::*;
     }
 }
+
+cfg_if! {
+    if #[cfg(feature = "extra_traits")] {
+        impl PartialEq for __c_anonymous_sigaction_handler {
+            fn eq(&self, other: &__c_anonymous_sigaction_handler) -> bool {
+                unsafe { self.default == other.default }
+            }
+        }
+        impl Eq for __c_anonymous_sigaction_handler {}
+        impl ::fmt::Debug for __c_anonymous_sigaction_handler {
+            fn fmt(&self, f: &mut ::fmt::Formatter) -> ::fmt::Result {
+                f.debug_struct("sigaction_t")
+                    .field("value", unsafe { &self.default } )
+                    .finish()
+            }
+        }
+        impl ::hash::Hash for __c_anonymous_sigaction_handler {
+            fn hash<H: ::hash::Hasher>(&self, state: &mut H) {
+                unsafe { self.default.hash(state) };
+            }
+        }
+    }
+}
```

Comment:
Could you move this before the consts declaration?

Reply:
Yep! Moved in 6913b31. Let me know if that's where you meant.